### ISSUE TITLE
Setup of Xdebug fails on `windows-latest` and PHP 8.1.17

### ISSUE THREAD

**Issue by localheinz** [shivammathur/setup-php#711]
**Describe the bug**

**Version**
- [x] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**
- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
- `windows-latest`

**PHP versions**
- 8.1.17

**To Reproduce**
See [`.github/workflows/ci.yml`](https://github.com/sebastianbergmann/php-code-coverage/blob/ec30451bb07c734000871c8ddf4445a176c4db2f/.github/workflows/ci.yml#L56-L107) in [`sebastianbergmann/php-code-coverage`](https://github.com/sebastianbergmann/php-code-coverage):

```yaml
# https://help.github.com/en/categories/automating-your-workflow-with-github-actions

on:
  - pull_request
  - push

name: CI

env:
  COMPOSER_ROOT_VERSION: "10.1-dev"

jobs:
  tests:
    name: Tests

    runs-on: ${{ matrix.os }}

    env:
      PHP_EXTENSIONS: dom, json, libxml, mbstring, pdo_sqlite, soap, xml, xmlwriter, :apcu, :imagick
      PHP_INI_VALUES: memory_limit=-1, assert.exception=1, zend.assertions=1, error_reporting=-1, log_errors_max_len=0, display_errors=On

    strategy:
      fail-fast: false
      matrix:
        os:
          - ubuntu-latest
          - windows-latest

        php-version:
          - "8.1"
          - "8.2"
          - "8.3"

        coverage-driver:
          - "pcov"
          - "xdebug3"

    steps:
      - name: Configure Git to avoid issues with line endings
        if: matrix.os == 'windows-latest'
        run: git config --global core.autocrlf false

      - name: Checkout
        uses: actions/checkout@v3

      - name: Install PHP with extensions
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php-version }}
          coverage: ${{ matrix.coverage-driver }}
          extensions: ${{ env.PHP_EXTENSIONS }}
          ini-values: ${{ env.PHP_INI_VALUES }}
          tools: none

      - name: Install dependencies with Composer
        run: php ./tools/composer update --no-ansi --no-interaction --no-progress

      - name: Run tests with PHPUnit
        run: vendor/bin/phpunit --coverage-clover=coverage.xml

      - name: Send code coverage report to Codecov.io
        uses: codecov/codecov-action@v3
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
```

**Expected behavior**
Set up PHP fails or Xdebug is installed

**Screenshots/Logs**
See https://github.com/sebastianbergmann/php-code-coverage/actions/runs/4447135572/jobs/7808303371#step:4:35

**Additional context**
Perhaps related to #700.

**Are you willing to submit a PR?**
n/a

**Comment by shivammathur** [comment]
Yes, it is related to #700, I will try to do a release this weekend.

**Comment by shivammathur** [comment]
Fix released in [`2.25.0`](https://github.com/shivammathur/setup-php/releases/tag/2.25.0).

**Comment by localheinz** [comment]
Thank you, @shivammathur!
